Dockerfile

```diff
@@ -18,13 +18,13 @@ ENV NEON_REVISION=${REVISION}
 RUN cargo fmt --check && \
     cargo clippy --release && \
     cargo build --release && \
-    cargo build-bpf --features devnet && cp target/deploy/evm_loader.so target/deploy/evm_loader-devnet.so && \
-    cargo build-bpf --features testnet && cp target/deploy/evm_loader.so target/deploy/evm_loader-testnet.so && \
-    cargo build-bpf --features govertest && cp target/deploy/evm_loader.so target/deploy/evm_loader-govertest.so && \
-    cargo build-bpf --features govertest,emergency && cp target/deploy/evm_loader.so target/deploy/evm_loader-govertest-emergency.so && \
-    cargo build-bpf --features mainnet && cp target/deploy/evm_loader.so target/deploy/evm_loader-mainnet.so && \
-    cargo build-bpf --features mainnet,emergency && cp target/deploy/evm_loader.so target/deploy/evm_loader-mainnet-emergency.so && \
-    cargo build-bpf --features ci --dump
+    cargo build-bpf --features devnet,maybe-async/is_sync && cp target/deploy/evm_loader.so target/deploy/evm_loader-devnet.so && \
+    cargo build-bpf --features testnet,maybe-async/is_sync && cp target/deploy/evm_loader.so target/deploy/evm_loader-testnet.so && \
+    cargo build-bpf --features govertest,maybe-async/is_sync && cp target/deploy/evm_loader.so target/deploy/evm_loader-govertest.so && \
+    cargo build-bpf --features govertest,emergency,maybe-async/is_sync && cp target/deploy/evm_loader.so target/deploy/evm_loader-govertest-emergency.so && \
+    cargo build-bpf --features mainnet,maybe-async/is_sync && cp target/deploy/evm_loader.so target/deploy/evm_loader-mainnet.so && \
+    cargo build-bpf --features mainnet,emergency,maybe-async/is_sync && cp target/deploy/evm_loader.so target/deploy/evm_loader-mainnet-emergency.so && \
+    cargo build-bpf --features ci,maybe-async/is_sync --dump
```

Comment:
Please include `maybe-async/is_sync` to default_features

Reply:
The `maybe_async/is-sync` feature cannot be added to the `default` features of `evm-loader` crate, since it cannot be disabled when building the crate as a library with `cargo build` (because of a Cargo bug, which [always enables default features for a workspace build](https://github.com/rust-lang/cargo/issues/10636)):
```
cargo build-sbf --manifest-path program/Cargo.toml --features maybe-async/is_sync
```

Reply:
Also, why did you make `maybe-async` optional in evm_loader?

Reply:
It was by accident, I made it not optional.